simpeg/directives/directives.py

```diff
@@ -347,16 +349,18 @@ class BaseBetaEstimator(InversionDirective):
     ----------
     beta0_ratio : float
         Desired ratio between data misfit and model objective function at initial beta iteration.
-    seed : int, None
-        Seed used for random sampling.
+    seed : {None, RandomSeed}, optional
```

Comment:
As we chatted about, I think it would be great to call this `random_seed` so it is clear to users that this is controlling the randomness, but no problem if this is addressed in a separate pr

Reply:
That sounds like a good name to me! that or just `rng` for random number generator?

Reply:
My vote would be for spelling it out `random_seed` (`rng` could easily be confused with `range` in some contexts)

Reply:
I agree with renaming it to `random_seed`. Is the name I used when I added the parameter to the [`make_synthetic_data`](https://docs.simpeg.xyz/content/api/generated/SimPEG.potential_fields.gravity.Simulation3DIntegral.make_synthetic_data.html#SimPEG.potential_fields.gravity.Simulation3DIntegral.make_synthetic_data) methods of simulations.

I'll open an issue for the renaming, so I can start working on that after this gets merged.